plugins/wp-songs-library/includes/class-wp-songs-library.php

```diff
@@ -270,6 +278,12 @@ private function define_custom_tables() {
 		$this->loader->add_action( 'plugins_loaded', $plugin_custom_table, 'register_custom_tables' );
 	}
 
+	private function define_cli_command() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this open curly brace to the beginning of the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://54.213.144.90:9000/sonar/coding_rules#rule_key=php%3AS1808)